src/main/java/tempest/Old/ModuleView.java

```diff
@@ -1,4 +1,8 @@
-package tempest;
+package tempest.Old;
```

Comment:
Why does this code need to stick around?

Reply:
Yeah it doesn't, I'll delete it now, but it was useful to refer to when doing some refactoring